### PR TITLE
moved search district to restaurants/index. Changed styling of homepage

### PR DESCRIPTION
**Pull request by tonianner**
moved search district to restaurants/index.html.erb
Changed styling of homepage
Authentication added for comment creation

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -95,14 +102,29 @@ input, button, select, textarea {
...
+
+@media (max-width:640px) {
+  .intro-header {
+    padding: 75px 0 25px 0;
+  }
+  .intro-header h1 {
+    font-size: 60px;
+  }
+  .intro-header h2 {
+    font-size: 30px;
```
Comment (dshcheung) [inline app/assets/stylesheets/application.scss]: please refactor this by nesting them

File: app/views/restaurants/index.html.erb [hunk 2]
```diff
@@ -2,11 +2,25 @@
...
 </div>
 
+<div class="container">
+<div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well">
+  <%= form_tag 'restaurants/search' do |f| %>
+    <div class="form-group">
+      <%= collection_select(:restaurant, :district_id, District.order(:name).all, :id, :name, {prompt: "Filter by distict"}, {class: "form-control"}) %>
+    </div>
+      <%= submit_tag 'Search', class:"btn btn-warning btn-block" %>
+      <%= link_to 'Show all', restaurants_path, class:"btn btn-primary btn-block" %>
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb:17]: indentations

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (tonianner) [inline app/assets/stylesheets/application.scss]: Oh yes! tnx :star2:

File: app/views/restaurants/index.html.erb (hunk 2 above)
Comment (tonianner) [inline app/views/restaurants/index.html.erb:17]: tnx.. sorry about that..

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -33,35 +35,40 @@ input, button, select, textarea {
   margin-right: 20px;
 }
 
-.home-header, .home-header2{
+.home_btn {
+  display: block;
+  border: 1px solid;
+  font-size: -webkit-xxx-large;
```
Comment (dshcheung) [inline app/assets/stylesheets/application.scss:41]: what is this `-webkit-xxx-large`?

Comment (tonianner) [inline app/assets/stylesheets/application.scss:41]: screen shot 2015-11-04 at 9 51 07 pm

In inspect element, it had that choice.

**Pull request merged**